src/main/webapp/app/shared/util/jhipster-types.ts

```diff
@@ -1,7 +1,7 @@
 import { AxiosResponse } from 'axios';
 import { CancellablePromise } from 'mobx/dist/api/flow';
 
-export type IPayload<T> = Promise<AxiosResponse<T>> | CancellablePromise<unknown>;
+export type IPayload<T> = Promise<AxiosResponse<T>> | CancellablePromise<AxiosResponse<T>>;
```

Comment:
Why did jhipster use `unknown` instead of `AxiosResponse<T>` before?

Reply:
No idea, but seems like type T is what should be expected based on the code I saw